USERS.md

```diff
@@ -108,6 +108,12 @@ Users (Alphabetically)
       U: ENI networking, L3/L4 policies, FQDN based policy, FQDN filtering
       Q: ungureanuvladvictor
       
+    * N: PostFinance AG 
+      D: PostFinance is using Cilium as their CNI for all mission critical, on premise k8s clusters 
+      U: Networking, network policies, kube-proxy replacement 
+      L: https://github.com/postfinance
+
+      
```

Comment:
Please would you mind removing the second blank line, to keep the style consistent with the rest of the file?